index.html

```diff
@@ -117,13 +134,13 @@
 		
 		
 		<script type="text/javascript">
-			function animate() {
-				Array.prototype.forEach.call(document.querySelectorAll('.willAnimate'), function(toAnimate, index) {
-					window.setTimeout(function() {
-						toAnimate.className = toAnimate.className.replace('willAnimate', 'animated')
-					}, 1000 + index * 500);
+            (function animate() {
+				Array.prototype.forEach.call(document.querySelectorAll('.animated'), function(toAnimate, index) {
+                    ["webkitAnimationDelay", "mozAnimationDelay", "animationDelay"].forEach(function(att) {
```

Comment:
Oh, really nice! I didn't know about `animation-delay`, thanks a lot! This could perhaps be applied through CSS for increased readability and total removal of JS.